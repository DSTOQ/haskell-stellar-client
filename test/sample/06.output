TransactionEnvelope
  { _transaction =
      Transaction
        { _sourceAccount =
            GAIH3ULLFQ4DGSECF2AR555KZ4KNDGEKN4AFI4SU2M7B43MGK3QJZNSR
        , _fee = FeeStroops { _feeStroops = 100 }
        , _seqNum =
            SequenceNumber
              { _sequenceNumber = NonNegativeInt64 (Refined 27797951757909318) }
        , _timeBounds = Nothing
        , _memo = MemoNone
        , _operations =
            [ Operation
                { _sourceAccount = Nothing
                , _body =
                    CreateAccount
                      CreateAccountOp
                        { _destination =
                            GDRC5LVW7Z2UTAUE4L6FRZ2J4LLRFZR2R66MO7GD5S73GTRPWNIBC4GF
                        , _startingBalance = Stroop 100000000000
                        }
                }
            ]
        }
  , _signatures =
      [ DecoratedSignature
          { _hint = SignatureHint { _signatureHint = "8656e09c" }
          , _signature =
              Signature
                { _signature =
                    "8efec69f3eca9f11e935b099aa6b23deef377f6e00c3e9eeb2ece8d44e258cfca811eeba5a7713a569d4d0db3c0e6279290f12a29139a070d388f4679cf79803"
                }
          }
      ]
  }