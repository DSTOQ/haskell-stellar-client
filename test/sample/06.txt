AAAAABB90WssODNIgi6BHveqzxTRmIpvAFRyVNM+Hm2GVuCcAAAAZABiwhcAA31GAAAAAAAAAAAAAAABAAAAAAAAAAAAAAAA4i6utv51SYKE4vxY50ni1xLmOo+8x3zD7L+zTi+zUBEAAAAXSHboAAAAAAAAAAABhlbgnAAAAECO/safPsqfEek1sJmqayPe7zd/bgDD6e6y7OjUTiWM/KgR7rpadxOladTQ2zwOYnkpDxKikTmgcNOI9Gec95gD
