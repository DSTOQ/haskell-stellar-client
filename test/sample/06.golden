TransactionEnvelope
  { _transaction =
      Transaction
        { _sourceAccount =
            PublicKeyEd25519
              { _publicKeyEd25519 =
                  [ 49
                  , 48
                  , 55
                  , 100
                  , 100
                  , 49
                  , 54
                  , 98
                  , 50
                  , 99
                  , 51
                  , 56
                  , 51
                  , 51
                  , 52
                  , 56
                  , 56
                  , 50
                  , 50
                  , 101
                  , 56
                  , 49
                  , 49
                  , 101
                  , 102
                  , 55
                  , 97
                  , 97
                  , 99
                  , 102
                  , 49
                  , 52
                  , 100
                  , 49
                  , 57
                  , 56
                  , 56
                  , 97
                  , 54
                  , 102
                  , 48
                  , 48
                  , 53
                  , 52
                  , 55
                  , 50
                  , 53
                  , 52
                  , 100
                  , 51
                  , 51
                  , 101
                  , 49
                  , 101
                  , 54
                  , 100
                  , 56
                  , 54
                  , 53
                  , 54
                  , 101
                  , 48
                  , 57
                  , 99
                  ]
              }
        , _fee = FeeStroops { _feeStroops = 100 }
        , _seqNum =
            SequenceNumber
              { _sequenceNumber = NonNegativeInt64 (Refined 27797951757909318) }
        , _timeBounds = Nothing
        , _memo = MemoNone
        , _operations =
            [ Operation
                { _sourceAccount = Nothing
                , _body =
                    CreateAccount
                      CreateAccountOp
                        { _destination =
                            PublicKeyEd25519
                              { _publicKeyEd25519 =
                                  [ 101
                                  , 50
                                  , 50
                                  , 101
                                  , 97
                                  , 101
                                  , 98
                                  , 54
                                  , 102
                                  , 101
                                  , 55
                                  , 53
                                  , 52
                                  , 57
                                  , 56
                                  , 50
                                  , 56
                                  , 52
                                  , 101
                                  , 50
                                  , 102
                                  , 99
                                  , 53
                                  , 56
                                  , 101
                                  , 55
                                  , 52
                                  , 57
                                  , 101
                                  , 50
                                  , 100
                                  , 55
                                  , 49
                                  , 50
                                  , 101
                                  , 54
                                  , 51
                                  , 97
                                  , 56
                                  , 102
                                  , 98
                                  , 99
                                  , 99
                                  , 55
                                  , 55
                                  , 99
                                  , 99
                                  , 51
                                  , 101
                                  , 99
                                  , 98
                                  , 102
                                  , 98
                                  , 51
                                  , 52
                                  , 101
                                  , 50
                                  , 102
                                  , 98
                                  , 51
                                  , 53
                                  , 48
                                  , 49
                                  , 49
                                  ]
                              }
                        , _startingBalance = Stroop 100000000000
                        }
                }
            ]
        }
  , _signatures =
      [ DecoratedSignature
          { _hint = SignatureHint [ 56 , 54 , 53 , 54 , 101 , 48 , 57 , 99 ]
          , _signature =
              Signature
                { _signature =
                    "[56,101,102,101,99,54,57,102,51,101,99,97,57,102,49,49,101,57,51,53,98,48,57,57,97,97,54,98,50,51,100,101,101,102,51,55,55,102,54,101,48,48,99,51,101,57,101,101,98,50,101,99,101,56,100,52,52,101,50,53,56,99,102,99,97,56,49,49,101,101,98,97,53,97,55,55,49,51,97,53,54,57,100,52,100,48,100,98,51,99,48,101,54,50,55,57,50,57,48,102,49,50,97,50,57,49,51,57,97,48,55,48,100,51,56,56,102,52,54,55,57,99,102,55,57,56,48,51]"
                }
          }
      ]
  }